cookbooks/bcpc-hadoop/libraries/utils.rb

```diff
@@ -391,7 +391,7 @@ def get_group_action(group_name)
 
 def has_vip?
   cmd = Mixlib::ShellOut.new(
-    "ip addr show dev #{node[:bcpc][:management][:interface]}", :timeout => 10
+    "ip addr", :timeout => 10
```

Comment:
I would suggest to use `ip addr list` instead of `ip addr` to be more explicit that we want a listing of all addresses.